docs/integration-tests.md

```diff
@@ -0,0 +1,27 @@
+---
+id: integration-tests
+title: Integration Test Docs
+slug: /integration-tests
+---
+
+# Integraiton Tests
+
+## About
+
+The integration test suite is an end-to-end test suite that uses Selenium and Firefox to verify nimbus functions properly for users.
+
+## Nimbus Tests
+### Getting Started
+
+1. Create a python virtual environment
```

Comment:
The make commands don't depend on a virtual environment, they depend on having Docker installed?

Reply:
Oof, good point